bedrock/base/templates/macros.html

```diff
@@ -35,15 +35,6 @@
   {% endfor %}
 {%- endmacro %}
 
-{% macro lazy_image(image_url, image_highres_url, placeholder_url, image_class) -%}
```

Comment:
Looks like this is still imported in `firefox/features/index.html`. It doesn't appear to be actually used in that file, and also doesn't cause an error as-is, but still probably best to remove the import. 😉